src/vmm/src/vmm_config/net.rs

```diff
@@ -116,140 +104,94 @@ impl Display for NetworkInterfaceError {
                     tap_err
                 )
             }
-            RateLimiterUpdateFailed(ref e) => write!(f, "Unable to update rate limiter: {:?}", e),
         }
     }
 }
 
-/// A wrapper over the list of the `NetworkInterfaceConfig` that the microvm has configured.
+type Result<T> = result::Result<T, NetworkInterfaceError>;
+
+/// A list of the Network Interfaces that the microvm has configured.
 #[derive(Default)]
-pub struct NetworkInterfaceConfigs {
-    if_list: Vec<NetworkInterfaceConfig>,
+pub struct NetworkInterfaces {
+    if_list: Vec<Arc<Mutex<Net>>>,
 }
 
-impl NetworkInterfaceConfigs {
-    /// Creates an empty list of NetworkInterfaceConfig.
+impl NetworkInterfaces {
```

Comment:
Seems that there is a bit of a difference between the interfaces and naming  of VsockStore, BlockDevices and NetworkInterfaces.

Reply:
That's right, I haven't paid much attention to naming consistency. I will rename all consistently, I'm thinking `Block/Net/Vsock Builder`.

Reply:
done